usabilla/php-docker-template
# Allow to customize `client_max_body_size`

Context
=======

This setting is essential for applications receiving higher payloads, uploads for instance.

How to test
=======

- `make build-http`
- `docker-compose up -d`
- Make a request with a size more than 1mb, see a 413 error
- Change your container setting to allow more
- Make a bigger request and see it works

A curl example with a file: 
```
curl -X POST -F 'data=@/home/renatomefi/Workspace/Usabilla/php-docker-template/tmp/17mb.input' http://localhost:8080
```

Generating a big file:
```
dd if=/dev/urandom of=./tmp/17mb.input bs=1MB count=17
```
